std/variant.d

```diff
@@ -1215,6 +1215,51 @@ public:
     }
 }
 
+///
+@system unittest
+{
+    Variant a; // Must assign before use, otherwise exception ensues
```

Comment:
Shouldn't these be examples of `VariantN`?

Reply:
GitHub's comment matching algorithm doesn't seem to be ideal :/
Anyhow, I now use `alias Var = VariantN!(maxSize!(int, real, double));` for the `VariantN` examples.